pkg/reconciler/openshift/rbac/rbac.go

```diff
@@ -274,3 +315,41 @@ func createSA(ctx context.Context, saInterface v1.ServiceAccountInterface, ns st
 
 	return sa, nil
 }
+
+func createTrustedCABundleConfigMap(ctx context.Context, cfgInterface v1.ConfigMapInterface, name, ns string) error {
+	c := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
```

Comment:
shall we add a prefix/suffix to this name. at present, this name is same as namespace name, isn't it.
it might be helpful to add a preffix/suffix, add an annotation to indicate who created it and for what purpose.

Reply:
> shall we add a prefix/suffix to this name. at present, this name is same as namespace name, isn't it.

right now the name of the configmaps are `config-service-cabundle` and `config-trusted-cabundle` which are sort of inline the configmap named `config-registry-cert` which lets users add their own certs - so it's not the same as namespace name :P

> it might be helpful to add a preffix/suffix, add an annotation to indicate who created it and for what purpose.

Sure, I'm adding `app.kubernetes.io/part-of: tekton-pipelines` label to these configmaps.

Reply:
oops, `serviceCABundleCofigMap` argument becomes the name, i was thinking `ns.Name` is used as the name :P